projects/core/src/routing/configurable-routes/config/routing-config.ts

```diff
@@ -2,6 +2,7 @@ import { RoutesConfig } from '../routes-config';
 
 export abstract class RoutingConfig {
   routing?: {
-    routes: RoutesConfig;
+    routes?: RoutesConfig;
+    protected?: boolean;
```

Comment:
I think `protected` option is not self-explanatory. Could you at least provide some comment for Intellisense?
Same applies to route config.